docs/troubleshooting/authentication/google_sign_in_troubleshooting.md

```diff
@@ -0,0 +1,67 @@
+---
+keywords: ['firebase', 'deployment', 'android']
+slug: /google-sign-in-troubleshooting
+title: Google Sign-In Troubleshooting
+---
+# Google Sign-In Troubleshooting
+
+If you face any issue while using the Google Sign-in feature from the exported app, then follow the given instructions to resolve them:
+
+1. **If the App is Pushed to Play Store from FlutterFlow by using CodeMagic deployment:**
```

Comment:
Please see our community guidelines and search for "bold". we dont use bold for complete sentences. Unless its a topic starter. In this case it sounds like a paragraph sentence.